google_compute_engine_oslogin/utils/oslogin_utils.h

```diff
@@ -79,6 +79,12 @@ class NssCache {
   // true on success.
   bool LoadJsonArrayToCache(string response);
 
+  // Helper method that effectively implements the getpwent_r nss method. Each call will
```

Comment:
Line length should be at most 80 characters long (unless I'm missing something).